src/foam/core/stdlib.js

```diff
@@ -548,7 +548,7 @@ foam.LIB({
       name: 'constantize',
       code: foam.Function.memoize1(function(/* String */ str) {
         // switches from from camelCase to CAMEL_CASE
-        return str.replace(/([a-z])([^0-9a-z_])/g, '$1_$2').toUpperCase();
+        return str.replace(/([a-z])([^0-9a-z_])/g, '$1_$2').replace(/\s/g,'').toUpperCase();
```

Comment:
Why/where is this needed?

Reply:
when constantizing wizardlet names for analytic events, we get event names like `LET_'S_ GET_ TO_ KNOW_ YOU_!_COMPLETE` (instead of `LET_'S_GET_TO_KNOW_YOU_!_COMPLETE`)